guilford_court_commitment.yaml

```diff
@@ -290,9 +247,12 @@ associationDefinitions:
       ol.id:
         type: "ol.id"
         transforms:
-        - !<transforms.HashTransform>
-          columns: ["COMMITTED FIRST NAME", "COMMITTED MIDDLE NAME", "COMMITTED LAST NAME", "COMMITTED NAME SUFFIX", "OFFENDER NC DOC ID NUMBER", "OFFENDER NC DOC ID NUMBER"]
-          hashFunction: "sha256"
+        - !<transforms.ConcatCombineTransform>
+          transforms:
+          - !<transforms.ColumnTransform>
+            column: "OFFENDER NC DOC ID NUMBER"
+          - !<transforms.ValueTransform>
+            value: "1"
     name: "sentencedwith"
 
   partof:
```

Comment:
It looks like here we used "relatedto" instead, sorry. Used only the personID and added a 3 Could we match that? https://github.com/Lattice-Works/Guilford-Co-NC/blob/master/guilford_inmates_dps.yaml#L893